src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -647,6 +647,22 @@ namespace nupic {
           void setSynPermConnected(Real setSynPermConnected);
 
           /**
+          Returns the maximum permanence amount a synapse can
+          achieve.
+
+          @returns real number of the max permanence amount.
+          */
+          Real getSynPermMax() const;
+          /**
+          Sets the maximum permanence amount a synapse can
+          achieve.
+
+          @param setSynPermCMax real number of the maximum permanence
+          amount that a synapse can achieve.
+          */
+          void setSynPermMax(Real setSynPermConnected);
```

Comment:
whoops - copy and pasted from the above. should I also change line 647 so the parameter doesn't have "set"?